go.mod

```diff
@@ -1,33 +1,32 @@
 module github.com/containers/kubernetes-mcp-server
 
-go 1.24.1
+go 1.24.10
```

Comment:
@matzew @Cali0707 is there a reason to update it 1.24.10? can we just keep v1.24.4 ?

Reply:
The latest Go version we have for the [tag](registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20) is go v1.24.4